<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskLogRecordDialog</class>
 <widget class="QDialog" name="TaskLogRecordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>关于</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/logo.png</normaloff>images/logo.png</iconset>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>450</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>40</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>案件名称：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_taskCollectMan">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>598</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>采集人员：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_taskAddress">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>189</y>
     <width>598</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>189</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>地点：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_taskDescript">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>294</y>
     <width>598</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>294</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>案件描述：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_taskText">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>400</y>
     <width>598</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>备注：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>131</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>时间：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_tasktime">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>131</y>
     <width>598</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_taskDeviceIP">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>346</y>
     <width>598</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>346</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>设备IP：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_taskDir">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>598</width>
     <height>31</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>案件目录：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_taskName">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>598</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
